h/static/scripts/group-forms/components/AnnotationCard.tsx

```diff
@@ -70,6 +77,19 @@ export default function AnnotationCard({
     [config.context.group],
   );
   const isReply = (annotation.references ?? []).length > 0;
+  const reloadAnnotation = useCallback(async () => {
+    if (!config.api.annotationDetail) {
+      return;
+    }
```

Comment:
`annotationDetail` is a required API property isn't it? In other words, there isn't an expected scenario where this API is missing and if we get here the property is missing, that's a bug in our code.

Reply:
In that case one option is to throw an error, to make it clear that such a situation represents programmer error rather than an expected situation that needs to be handled. You can mark the branch as skipped from test coverage.

Reply:
An even simpler option is to use `!` to indicate that the route is expected to be available here.

Reply:
Yeah, good point. I'll do that.